chef/automate
# Installing profiles to the compliance-service

### :nut_and_bolt: Description: What code changed, and why?
Finishing the functionality of when a user clicks the "install" button on a profile content item. When this happens the profile is installed for the current login in user only. This add the compliance-service as a dependency on the automate-cds service. 

### :chains: Related Resources

https://github.com/chef/automate/issues/3900

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-deployment && build components/automate-gateway && build components/automate-cds && build components/automate-ui-devproxy && start_all_services && start_automate_ui_background && ui_logs`
1. After the UI finishes starting log in as admin and go to the profiles page (https://a2-dev.test/compliance/compliance-profiles).
1. Ensure there are no profiles install. 
1. Go to the CDS page (https://a2-dev.test/cds). 
1. Click "Install" on one or more profile content items. 
1. Ensure the success banner is displayed. 
1. Go back to the profiles page (https://a2-dev.test/compliance/compliance-profiles) and ensure those profiles are installed. 

### :white_check_mark: Checklist
